Cargo.lock

```diff
@@ -223,6 +223,13 @@ version = "0.1.17"
 source = "git+https://github.com/browserhtml/browserhtml?branch=crate#aafbb0996b02d1fadd4713c96e3d22b542f175a1"
 
 [[package]]
+name = "build-apk"
```

Comment:
This was a bug unrelated to this PR, it’s fixed on master now. This should disappear if you rebase.